### DISCUSSION TITLE
Cannot install Tempest addon

### DISCUSSION THREAD

**Discussion by Carib909** [community/community#141996] [Repositories]
When I install Tempest Repo and Temptv repo from Repository on Kodi 21.1, I get an error “Cannot connect to repository “. What’s the problem and how to fix it? Thanks.

**Comment by Sanidhyafeaturist** [comment]
It typically indicates a connectivity issue or an outdated repository URL. To resolve this, first ensure that your internet connection is stable. Next, check if the repository URLs are correct and up to date; outdated URLs can lead to connectivity failures. You can do this by removing the existing repository source and re-adding it with the latest link from the official source or community forums. Additionally, consider clearing the Kodi cache or reinstalling Kodi if issues persist. Finally, if the repository is currently down or the developers have ceased maintenance, you may need to look for alternative repositories or addons that offer similar functionality.

**Comment by novium2** [comment]
Hi @Carib909 , I'm moving this into the repos category. Just an FYI - posting in the dedicated category helps people find your post! :)

**Comment by Carib909** [comment]
Thanks. I just didn’t know where to post it.

Lester E. Flax


Semper Virens
